coral/src/app/features/approvals/acls/components/AclApprovalsTable.tsx

```diff
@@ -0,0 +1,159 @@
+              {aclRequest.acl_ssl?.map((ssl, index) => (
+                <StatusChip
+                  status="neutral"
+                  key={`${ssl}-${index}`}
+                  text={`${ssl} `}
+                  dense
+                />
+              ))}
+            </Table.Cell>
+            <Table.Cell>
+              {aclRequest.acl_ip?.map((ip, index) => (
+                <Flexbox
+                  key={`${ip}-${index}`}
+                  gap={"2"}
+                  wrap="wrap"
+                  direction={"row"}
+                >
+                  <StatusChip status="neutral" text={`${ip} `} dense />
+                </Flexbox>
+              ))}
+            </Table.Cell>
+            <Table.Cell>
+              {aclRequest.topicname}
+              {aclRequest.aclPatternType === "PREFIXED" && (
+                <code>(prefixed)</code>
+              )}
+            </Table.Cell>
+            <Table.Cell>
+              <StatusChip
+                status="neutral"
+                text={aclRequest.environment ?? "-"}
+                dense
+              />
+            </Table.Cell>
+            <Table.Cell>{aclRequest.teamname}</Table.Cell>
+            <Table.Cell>
+              <StatusChip
+                status={aclRequest.aclType === "CONSUMER" ? "success" : "info"}
+                text={aclRequest.aclType}
+                dense
+              />
+            </Table.Cell>
+            <Table.Cell>
+              {!aclRequest.requestStatus ? (
+                <StatusChip status="neutral" text={"-"} dense />
+              ) : (
+                <StatusChip
+                  status={requestStatusChipStatusMap[aclRequest.requestStatus]}
+                  text={requestStatusNameMap[aclRequest.requestStatus]}
+                  dense
+                />
+              )}
+            </Table.Cell>
+            <Table.Cell>{aclRequest.username}</Table.Cell>
+            <Table.Cell>{aclRequest.requesttimestring ?? "-"} UTC</Table.Cell>
+            <Table.Cell>
+              <GhostButton
+                title="View request details"
+                aria-label={`View topic request for ${aclRequest.topicname}`}
+                icon={infoSign}
+                onClick={() => onDetails(aclRequest)}
+                dense
+              >
+                <span aria-hidden={"true"}>View details</span>
+              </GhostButton>
+            </Table.Cell>
+            <Table.Cell>
+              {aclRequest.requestStatus === "CREATED" && (
+                <GhostButton
+                  title={"Approve acl request"}
+                  aria-label={`Approve schema request for ${aclRequest.topicname}`}
+                  dense
+                  onClick={() => {
+                    setLoading(true);
+                    onApprove(aclRequest);
+                  }}
+                >
+                  <Icon
+                    color="grey-70"
+                    icon={isLoading ? loadingIcon : tickCircle}
```

Comment:
Unfortunately, relying on that state to show the loading icon will also show a loading icon for all rows when clicking this button ^^' This is why we *also* rely on the `useMutation`'s `isLoading` return value as well to correctly render this loading state in the other Approval tables.


https://user-images.githubusercontent.com/20607294/222657403-4fd655f9-7162-4ce0-9934-f3686ce3839e.mov


To be noted this double dependency on a local `isLoading` state + the `isLoading` value from `useMutation` is pretty icky, and there is an issue to address it (among other icky stuff): https://github.com/aiven/klaw/issues/736